ansible/roles/builds/tasks/main.yaml

```diff
@@ -2,23 +2,23 @@
 - name: Create Host OS build directories
```

Comment:
Explain in the commit message that "This improves isolation of the different user roles in a setup that creates master and slave in the same machine."